tests/failing.nix

```diff
@@ -0,0 +1,22 @@
+{ busybox }:
+with import ./config.nix;
+let
+
+  mkDerivation = args:
+    derivation ({
+      inherit system;
+      builder = busybox;
+      args = ["sh" "-e" args.builder or (builtins.toFile "builder-${args.name}.sh" "if [ -e .attrs.sh ]; then source .attrs.sh; fi; eval \"$buildCommand\"")];
+    } // removeAttrs args ["builder" "meta"])
+    // { meta = args.meta or {}; };
```

Comment:
Why do you need all that in addition to the `mkDerivation` that `config.nix` already exports?

Reply:
The test otherwise fails with

```
    copying 1 paths...
    copying path '\''/nix/store/szbwpiamb2znfgw6iv8c8rr7m5d2b9vz-builder-failing.sh'\'' to '\''ssh://localhost'\''...

    ErroSysError~executing '\''/nix/store/xvvgw9sb8wk6d2c0j3ybn7sll67s3s4z-bash-4.4-p23/bin/bash'\'': No such file or directory
    note: keeping build directory '\''/tmp/nix-shell.2zeviF/nix-build-failing.drv-3'\''
    error: build of '\''/nix/store/i3a2nqjbapjrspr1n1q7f9mmwivdcva6-failing.drv'\'' on '\''ssh://localhost?remote-store=/tmp/nix-shell.2zeviF/nix-test/tests/build-remote-input-addressed/machine1?system-features=foo'\'' failed: error: builder for '\''/nix/store/i3a2nqjbapjrspr1n1q7f9mmwivdcva6-failing.drv'\'' failed with exit code 1
    error: builder for '\''/nix/store/i3a2nqjbapjrspr1n1q7f9mmwivdcva6-failing.drv'\'' failed with exit code 1'
```

All of the `build-remote.sh` tests seem to get their own `$busybox` at the moment and this test uses it rather than `shell` as builder.